lib/limits.c

```diff
@@ -134,9 +135,16 @@ static int check_logins (const char *name, const char *maxlogins)
 		return LOGIN_ERROR_LOGIN;
 	}
 
-	count = active_sessions_count(name, limit);
+#ifdef ENABLE_LOGIND
+	if (exceeded < 0)
```

Comment:
Isn't this dead code?  The condition is always true, because exceeded is `-1`, so you could do that unconditionally, right?